scripts/collect-auction-item-candidates.ts

```
@@ -0,0 +1,160 @@
+import "dotenv/config";
+
+import { mkdirSync, readFileSync, writeFileSync } from "node:fs";
+import { basename, resolve } from "node:path";
+import * as z from "zod";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)